src/conversion/view-selection-to-model-converters.js

```diff
@@ -39,10 +39,8 @@ export function convertSelectionChange( modelDocument, mapper ) {
 
 		modelSelection.setRanges( ranges, viewSelection.isBackward );
 
-		if ( !modelSelection.isEqual( modelDocument.selection ) ) {
```

Comment:
Maybe we should try to render in the selection observer if there are no changes? I remember there was a ticket about it...